### PR TITLE
Feature 1: user opens the program

### PR DESCRIPTION
**Pull request by stephaniesaab**
Created the ToDoList class with its private attribute (taskList) and its two public methods (addTask and removeTask). Javadocumentation for the class, its attribute and each method is added as well.

### CODE REVIEW

File: src/main/java/Entities/ToDoList.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package Entities;
```
Comment (harshsbajwa) [inline src/main/java/Entities/ToDoList.java:1]: We decided that entities would be lowercase (for consistency and in accordance with the demo repository), please change

**Review by harshsbajwa — CHANGES_REQUESTED** [review]
Please change "Entities" to "entities" for consistency

File: src/main/java/Entities/DoneList.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    //A public class that contains a list of tasks that the user has completed. Tasks are not sorted in this list.
+    //Tasks can be added to this list only if it is first checked off the checklist and removed from the ToDoList.
+    //Tasks can also be removed from this list if the user wants to add it to its
+
+    private List<Task> doneTasks; //A list of tasks completed by the user.
+
+    /**
+     * Initialize an empty DoneList with no tasks in its doneTasks list.
+     */
+    public DoneList() {
```
Comment (XinyingXu) [inline src/main/java/Entities/DoneList.java:16]: You could add another constructor which takes in a list as parameters, which could be used later by use cases.

File: src/main/java/Entities/ToDoList.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    }
+
+    /**
+     * Remove the task with the given name from this
+     * @param oldTask
+     */
+    public void removeTask(Task oldTask) {
+        this.taskList.remove(oldTask);
+    }
+
```
Comment (XinyingXu) [inline src/main/java/Entities/ToDoList.java:35]: My use case might need a searchFor(String taskName) method to locate a task in the list and return the Task object.

**Review by serenahonekin — CHANGES_REQUESTED** [review]
Could you also sort the ToDoList by the task's deadline?

**Review by e-laflamme — APPROVED** [review]
Looks good! You may want to leave out the gradle files when committing in the future though. 
Before merging, you should probably change the entities folder name and import statements to lowercase.

**Pull request merged**